src/lib/OpenEXRUtil/ImfCheckFile.cpp

```diff
@@ -736,10 +737,16 @@ readMultiPart(MultiPartInputFile& in,bool reduceMemory,bool reduceTime)
         //
         if (isTiled(in.header( part ).type()))
         {
-            if ( in.header( part ).tileDescription().ySize *  (b.max.x-b.min.x+1) > gMaxTilePixelsPerScanline )
+            const TileDescription& tileDescription = in.header( part ).tileDescription();
+            int tilesPerScanline = (b.max.x-b.min.x + tileDescription.xSize) / tileDescription.xSize;
+            if ( tileDescription.ySize * tileDescription.xSize * tilesPerScanline > gMaxTilePixelsPerScanline )
```

Comment:
Is fuzz testing going to next complain about tiles whose area is greater than 2^31 pixels? ;)